easybuild/easyblocks/a/abaqus.py

```diff
@@ -78,6 +79,9 @@ def extract_step(self):
     def configure_step(self):
         """Configure ABAQUS installation."""
         if LooseVersion(self.version) >= LooseVersion('2016'):
+            # Rocky Linux isn't recognized; faking it as RHEL
+            if get_os_name() == 'Rocky Linux':
```

Comment:
we might move to AlmaLinux, would be nice if that's supported too:
```suggestion
            if get_os_name() in ['Rocky Linux', 'AlmaLinux']:
```